consensus/proto_array/src/proto_array.rs

```diff
@@ -1055,13 +1055,7 @@ pub fn calculate_committee_fraction<E: EthSpec>(
     justified_balances: &JustifiedBalances,
     proposer_score_boost: u64,
 ) -> Option<u64> {
-    let average_balance = justified_balances
-        .total_effective_balance
-        .checked_div(justified_balances.num_active_validators)?;
-    let committee_size = justified_balances
-        .num_active_validators
-        .checked_div(E::slots_per_epoch())?;
-    let committee_weight = committee_size.checked_mul(average_balance)?;
+    let committee_weight = justified_balances.total_effective_balance;
```

Comment:
this is missing the division by slots per epoch, isn't it?

Reply:
correct, thanks for catching that